src/Services/Settings.js

```diff
@@ -14,7 +14,8 @@ const initialSettings = {
     network_id: 5777,
     total_accounts: 10,
     unlocked_accounts: [],
-    mnemonic: "candy maple cake sugar pudding cream honey rich smooth crumble sweet treat"
+    mnemonic: "candy maple cake sugar pudding cream honey rich smooth crumble sweet treat",
+    vmErrorsOnRPCResponse: true
```

Comment:
Isn't true the default value here?

Reply:
We discussed this offline, but just capturing here for completeness.

Yes, but since we can't read that out of ganache-core we have to explicitly define a default here. Otherwise the initial value shown by the UI will be false, as it'll interpret `undefined` falsely.